cms/static/js/hera/utils/partial.jsx

```diff
@@ -0,0 +1,7 @@
+export default function partial(f) {
```

Comment:
You do not use it, please remove

Reply:
Will you address this one?

Reply:
I left it so far, maybe it will be helpful in the future, if not I will delete it at all.